langchain/document_loaders/trello.py

```diff
@@ -0,0 +1,124 @@
+"""Loader that loads cards from Trello"""
+from typing import List, Optional
+
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseLoader
+
+class TrelloLoader(BaseLoader):
+    """Trello loader. Reads all cards from a Trello board.
+
+    Args:
+        api_key (str): Trello API key.
+        api_token (str): Trello API token.
+    """
+
+    def __init__(self, api_key: str, api_token: str):
+        """Initialize Trello loader."""
+        self.api_key = api_key
+        self.api_token = api_token
+
+    def load(
+        self,
```

Comment:
@GDrupal Thanks for the contribution! 

Document loaders require an argumentless load method.

Could you push the configuration into the initializer?

I've commented on the types of some of the parameters in `load`, so we can update the type signatures as well when moving to __  init __